Application/ControlMsrE2/ControlMsrE2.c

```diff
@@ -0,0 +1,242 @@
+/** @file
+ Control CFGLock BIOS Option
+ 
+ Copyright (c) 2020, Brumbaer. All rights reserved.<BR>
+ This program and the accompanying materials
+ are licensed and made available under the terms and conditions of the BSD License
+ which accompanies this distribution.  The full text of the license may be found at
+ http://opensource.org/licenses/bsd-license.php
+ 
+ THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+ 
+ **/
+#include "ControlMsrE2.h"
+
+#define CONTEXTS_MAX 8
+
+EFI_STATUS WalkListHeaders (
+                            IN EFI_HII_HANDLE * handles,
+                            IN EFI_HII_PACKAGE_LIST_HEADER **ListHeaders,
+                            IN UINT32 ListHeaderCount,
+                            IN EFI_STRING SearchString
+                            ) {
+    UINT16           OptionsCount;
```

Comment:
The indentation of the code should be made with **2 spaces**, and the function header should be formatted as follows:

```C
EFI_STATUS
WalkListHeaders (
  IN EFI_HII_HANDLE                *Handles,
  IN EFI_HII_PACKAGE_LIST_HEADER   **ListHeaders,
  IN UINT32                        ListHeaderCount,
  IN EFI_STRING                    SearchString
  )
{
  /* … */
}
```

Note the **newlines** as well as argument **indentation** and **alignment**. Also braces and curly-braces positioning.